ssr-express/README.md

```diff
@@ -0,0 +1,81 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Introduction
+
+amp-ssr-express is an [express](http://expressjs.com/) middleware that optimizes page load
+times for websites using AMP for their canonical pages. The middleware uses the same
+server-side-rendering optimizations as the Google AMP Cache.
+
+The middleware uses the [amp-ssr](../ssr) component to apply server-side-rendering on the fly.
+
+## How it works
+
+amp-ssr-express intercepts the responses and replaces their content with a version that has been
+transformed by [amp-ssr](../ssr).
+
+As the server-side-rendered version of the content is not valid AMP, the component also
+provides the original content on an alternative URL. Server-side-rendering
+transformations are not applied to this URL, and the original valid AMP is served.
+
+A amphtml link tag is added to the server-side-rendered version, linking it to the original valid
+AMP version hosted on the alernative URL.
+
+Example:
+
+A valid AMP page is served on `https://example.com/index.html`.
+
+When the amp-ssr-express middleware is used, that URL will serve the server-side-rendered version
+of the content.
+
+The original, valid AMP will then become available at `https://example.com/index.html?amp`.
+
+An amphtml link will be added to the server-side-rendered version:
+`<link rel="amphtml" href="https://example.com/index.html?amp">`.
+
+## Usage
+
+The AMP SSR middleware can be used like any other express middleware.
+
+It is important that the middleware is used *before* the middleware or route that renders the page.
+
+The example bellow will transform HTML being loaded by express-static:
+
+```javascript
+const express = require('express');
+const path = require('path');
+const app = express();
+const createAmpSsrMiddleware = require('amp-toolbox-ssr-express');
+const ampSsr = require('amp-toolbox-ssr');
```

Comment:
we shouldn't require to pass an amps instance

Reply:
done.